tools/iop_dependencies.py

```diff
@@ -369,6 +369,10 @@ def add_edges(gr):
   # want to change exposure/basecurve after tone mapping
   gr.add_edge(('exposure', 'tonemap'))
   gr.add_edge(('basecurve', 'tonemap'))
+  # fbw goes after tonemap but before exposure/basecurve
+  gr.add_edge(('freaky_bw', 'tonemap'))
```

Comment:
I wonder, if this is the right place in the pixel pipe. Isn't this too early? I had expect to find it much later, e.g., after the 'monochrome' module, in particular after exposure and basecurve.

Reply:
I tried several places in the pipe, this is what I find to be the best.